packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
+   selectedCategories TEXT,
+   graphType TEXT DEFAULT 'BarGraph',
+   viewLegend INTEGER DEFAULT 0,
+   viewSummary INTEGER DEFAULT 0,
+   viewLabels INTEGER DEFAULT 0,
+   conditions TEXT,
+   conditions_op TEXT DEFAULT 'and',
+   sort_order REAL,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think we'll want to create a completely new table for managing the `homepage` state, so I'd recommend not having this here.

For homepage (totally not for this PR and don't feel pressured to do the implementation either): 
- we would want to sort the graphs (not just custom reports, but also the generic ones)
- we would want to resize them (1x1; 1x2; 2x2; etc)
- we would want to add other widgets too (for example: text labels so it's easier to group your widgets)

It would also be super cool to be able to export ALL of the configuration (homepage layouts + widget definitions). That way the community could share them. But that's me just dreaming about the long-long term future :)